generate.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-while getopts g:t:m:u:w: option
+while getopts g:t:d:m:u:w: option
 do
  case "${option}"
  in
  g) GITURL=${OPTARG};;
  t) export DOCTHEME=${OPTARG};;
+ d) export DEBUG=${OPTARG};;
```

Comment:
what about its hindrance in concurrent executions? If one script execute this line and after some time another instance executes this without the completion of first, won't it lead to some problem?

I am not sure about this. It is just a concern